solutions/standard/README.md

```diff
@@ -0,0 +1,94 @@
+# VSI on VPC IBM Secure Landing Zone - Standard Variation
+
+[![Graduated (Supported)](https://img.shields.io/badge/status-Graduated%20(Supported)-brightgreen?style=plastic)](https://terraform-ibm-modules.github.io/documentation/#/badge-status)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![latest release](https://img.shields.io/github/v/release/terraform-ibm-modules/terraform-ibm-zvsi?logo=GitHub&sort=semver)](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/releases/latest)
+[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
+
+<!-- BEGIN MODULE HOOK -->
+
+![Architecture diagram for the Standard variation of VSI on VPC landing zone](https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-zvsi/standard_quickstart/reference-architecture/Standard-variation.svg)
+
+This pattern deploys the following infrastructure:
+
+- A resource group for cloud services and for each VPC.
+- Cloud Object Storage instances for flow logs and Activity Tracker.
+- Encryption keys in a Key Protect instance.
```

Comment:
Is "Key Protect" the correct term here?   From the terraform, it looks like you are using or provisioning a Secrets Manager instance.